app/src/main/java/unipd/se18/ocrcamera/CameraActivity.java

```diff
@@ -0,0 +1,123 @@
+/*
+ *  The main goal for this project is to test older Camera API (15 to 19) to see if it's
+ *  plausible to support these devices and lower the minimum API requirement on CommonDemo
+ *  so we can support full range of Android devices (API 15 and higher is 100% demographic)
+ *  I used this interface for Camera (1) API to make the program
+ *      https://github.com/natario1/CameraView
+ *  working with master build of the interface
+ *  The documentation starts here
+ *      https://github.com/natario1/CameraView#usage
+ */
+
+package com.example.elia.compatibilitycamera;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.view.View;
+import android.content.pm.PackageManager;
+import android.support.annotation.NonNull;
+import android.widget.ImageButton;
+
+import com.otaliastudios.cameraview.CameraListener;
+import com.otaliastudios.cameraview.CameraView;
+import com.otaliastudios.cameraview.Gesture;
+import com.otaliastudios.cameraview.GestureAction;
+
+/**
+ * Activity for camera preview and taking photos
+ */
+public class CameraActivity extends AppCompatActivity {
+
+    private CameraView camera;
+    private boolean mCapturingPicture;
```

Comment:
Is it a good idea to explicitly set this variable to false and put a short description about the future use of this?
But firstly is it a good idea to disable and enable the functionality of the button instead a boolean system?

Reply:
You are absolutely right, thank you.
I'll add descriptions and set the variable mCapturingPicture to default 'false'.

> But firstly is it a good idea to disable and enable the functionality of the button instead a boolean system?
See https://github.com/PrandiniUniPD/OCRCamera/pull/64#discussion_r238099864

Reply:
Ok, so if you think that it's a good idea enabling and disabling the button and you need some support ask me.

Reply:
Thank you very much, I'll keep that in mind!